### PR TITLE
Add loki log transport

### PR DESCRIPTION
**Pull request by apmorton**
May want to wait on this for https://github.com/JaniAnttonen/winston-loki/issues/47

Also need to packer and terraform a new AMI before deploying this.

### CODE REVIEW

File: lib/logger.js [hunk 1]
```diff
@@ -103,5 +121,5 @@ export function logToPapertrail(host, port, identifier) {
 }
 
 export function suppressConsoleLog() {
-    consoleTransportInstance.silent = true;
+    logger.remove(consoleTransportInstance);
```
Comment (mattgodbolt) [inline lib/logger.js:124]: much nicer

**Review by mattgodbolt — APPROVED** [review]
Looks good to go: we do'nt have to configure it on yet, we could wait for JaniAnttonen/winston-loki#47 (or point at a patched version?)

**Comment by mattgodbolt** [comment]
Looks like the loki thing was merged

**Comment by mattgodbolt** [comment]
> Also need to packer and terraform a new AMI before deploying this.

What's the change on that - do these all need to be done together, or can we update those first?

**Comment by apmorton** [comment]
> > Also need to packer and terraform a new AMI before deploying this.
> 
> What's the change on that - do these all need to be done together, or can we update those first?

If memory serves I think it was just this:
https://github.com/compiler-explorer/infra/commit/43ac626e7ab653427451b2187a643858ab2e57d6

At the time I wrote the note in the PR I had just committed that and not done packer/etc - if it has been done for any other reason since then I think its all set.

**Comment by mattgodbolt** [comment]
Confirmed this change is in packer etc and in the current images :)

**Pull request merged**